tests/std/tests/P0355R7_calendars_and_time_zones_io/test.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <chrono>
+#include <locale>
+#include <sstream>
+
+using namespace std;
+using chrono::duration;
+
+template <class CharT, class Rep, class Period>
+bool test_duration_basic_out(const duration<Rep, Period>& d, const CharT* expected) {
+    basic_ostringstream<CharT> ss;
+
+    ss << d;
+    return ss.str() == expected;
+}
+
+#define WIDEN(TYPE, STR) get<const TYPE*>(make_pair(STR, L##STR));
+
+template <class CharT>
+bool test_duration_locale_out() {
+    basic_stringstream<CharT> ss;
+    const duration<float> d{0.140625};

```

Comment:
`0.140625` is `double` which makes me worry about narrowing warnings to `float` (even though it's exactly representable in both types). An `f` suffix would put my mind at ease. Occurs below.

Reply:
Actually, I'd like to change this to `duration<double>`, since nothing about the narrowness of `float` matters for this test.